### ISSUE TITLE
The reverse command always says: No later change info

### ISSUE THREAD

**Issue by JSpenced** [emacs-evil/goto-chg#7]
Whenever I `goto-last-change` once or multiple times then try to run `goto-last-change-reverse`, I get the error: `No later change info`.

This can be fixed if I pass a prefix argument `C-u 0` before running the commands. Then both `goto-last-change` and `goto-last-change-reverse` work correctly.

I plan to just write a function to always call it with `C-u 0` or could this bug possibly be fixed. Let me know if you need any more information.

**Comment by wasamasa** [comment]
What you propose is a workaround, not a fix. If you understand the involved code to suggest a fix, I'm all ears.

**Comment by JSpenced** [comment]
I solved it doing this, which seems to work in most cases and works better than always not working. It just always passes in '-1 to the command so if you want to do like '-4 to go back 4 arguments then it won't work. But I can't see many use cases where you want to do like `Ctrl-u` then goto-last-change-reverse to go back 4 changes. It could be updated to do that, but I never use it in that manner.


```
(defun goto-last-change-reverse ()
  "Fix goto-last-change-reverse code that doesn't work properly"
  (interactive)
  ;; Make 'goto-last-change-reverse' look like 'goto-last-change'
  (cond ((eq last-command this-command)
	 (setq last-command 'goto-last-change)))
  (setq this-command 'goto-last-change)
  ;; FIXME: Fix so can pass in any number of previous argument
  (goto-last-change '-1))
```

**Comment by wasamasa** [comment]
Sorry, but I definitely won't apply that as is. If it works this way for you, feel free to apply it yourself by wrapping it into an `(with-eval-after-load 'goto-chg ...)`.

**Comment by JSpenced** [comment]
Understandable, yeah I just made a new function and call that instead of your version. Does the current version work at all for you? For me it doesn't, so at least this will go back one by one everytime.

**Comment by wasamasa** [comment]
It isn't indeed, but I don't see much motivation in comitting a still broken version into the repo.

**Comment by JSpenced** [comment]
Yeah, no worries. If there is a fix, at least post here so I can revert. I don't want to spend the time to make it work with any universal-argument inputs like it is trying to do now. I don't see the point of that and don't know well how to fix that. I just used the function I posted above and then wrote a function to go back to the most recent change. Normally, I want to go back one or all the way to the beginning. Thanks for responding.
